pootle/static/js/admin/components/Project/ProjectForm.js

```diff
@@ -78,13 +78,14 @@ const ProjectForm = React.createClass({
           />
           <FormElement
             type="select"
+            multiple
             clearable={false}
-            options={model.getFieldChoices('localfiletype')}
-            label={gettext('File Type')}
+            options={model.getFieldChoices('filetypes')}
+            label={gettext('Filetypes')}
```

Comment:
Not gonna bikeshed, but as a localizer I don't recall having to translate _filetype_, as opposed to _file type_. Feel free to check any online glossaries available out there (Mozilla, Microsoft...).